index.js

```diff
@@ -125,12 +125,27 @@ module.exports = function (message, options) {
       }, maxLength);
 
       if (index === 0) {
+        /**
+         * Need to adjust the top position of the speech bubble depending on the
+         * amount of lines of the message.
+         */
         if (array.length === 2) {
           topOffset -= 1;
         }
         if (array.length >= 3) {
           topOffset -= 2;
         }
+        /**
+         * The speech bubble will overflow the Yeoman character id the message
+         * has more than 7 lines. So we vertically center the bubble by adding
+         * empty lines on top of the greeting.
+         */
+        if (array.length > 7) {
+          for (var i = 0; i < Math.floor((array.length - 7) / 2); i++) {
```

Comment:
I did that. But im unsere var to place that variable declaration. Since we have no block level scope in ES5 😬 

I placed it directly outside the loop. in the block. Would you love to see it in [L80](https://github.com/yeoman/yosay/pull/23/files#diff-168726dbe96b3ce427e7fedce31bb0bcL82) instead